sources/odoo.conf

```diff
@@ -1,104 +1,9 @@
 [options]
-admin_passwd = _ADMIN_PASSWORD_
+admin_passwd = admin
 
-; set the default timezone for openerp interface
-timezone = Europe/Paris
-
-db_maxconn = 64
 db_name = False
-db_template = template1
-dbfilter = _DB_FILTER_
-
-; set this to True if you have installed unaccent in your PostgreSQL
-; and want to be able to search without accents
-unaccent = True
-
-; These options are really important and depend on the type of
-; deployment you want to have so read the doc carefully and
-; set these values accordingly
-workers = 0
-max_cron_threads = 2
-limit_memory_hard = 2147483648
-limit_memory_soft = 671088640
-limit_request = 4096
-limit_time_cpu = 120
-limit_time_real = 180
-
-; -----------------------------------------------------------------
-; Below options are less critical but you are welcome to tweak them
-; -----------------------------------------------------------------
-
-csv_internal_sep = ,
-
-debug_mode = False
-demo = {}
-
-email_from = False
-import_partial = 
-
-list_db = True
-
-log_handler = [':INFO']
-log_level = info
-logfile = None
-
-login_message = False
-
-logrotate = True
-
-netrpc = False
-netrpc_interface = 
-netrpc_port = 8070
-
-osv_memory_age_limit = 1.0
-osv_memory_count_limit = False
-
-pg_path = None
-pidfile = None
-
-proxy_mode = False
-reportgz = False
-
-secure_cert_file = server.cert
-secure_pkey_file = server.pkey
-
-server_wide_modules = None
-
-smtp_password = False
-smtp_port = 25
-smtp_server = localhost
-smtp_ssl = False
-smtp_user = False
-
-static_http_document_root = None
-static_http_enable = False
-static_http_url_prefix = None
-
-syslog = False
-
-test_commit = False
-test_enable = False
-test_file = False
-test_report_directory = False
-
-translate_modules = ['all']
-without_demo = False
-
-xmlrpc = True
-xmlrpc_interface =
-xmlrpc_port = 8069
-xmlrpcs = True
-xmlrpcs_interface = 
-xmlrpcs_port = 8071
-
-; DO NOT CHANGE THIS... It is setup accordingly to internal docker image layout
-; READ the example run script to understand how it works
-
-; do not change this
 db_host = db
-; if False then Odoo will fallback to 5432
 db_port = False
 
-; the PostgreSQL connection role Odoo will use to connect to its databases
-db_user = _DB_USER_
-db_password = _DB_PASSWORD_
+timezone = Etc/UTC
+proxy_mode = False
```

Comment:
I removed the parameter to use Odoo's default value which should be `False` ([the documentation](https://www.odoo.com/documentation/8.0/reference/cmdline.html#running-the-server) doesn't contain the info)